tests/e2e/General/UsageTest.php

```diff
@@ -795,15 +795,17 @@ public function testFunctionsStats(array $data): void
         );
 
         $this->assertEquals(200, $response['headers']['status-code']);
-        $this->assertEquals(17, count($response['body']));
+        $this->assertEquals(21, count($response['body']));
         $this->assertEquals($response['body']['range'], '30d');
         $this->assertIsArray($response['body']['functions']);
         $this->assertIsArray($response['body']['deployments']);
         $this->assertIsArray($response['body']['deploymentsStorage']);
         $this->assertIsArray($response['body']['builds']);
         $this->assertIsArray($response['body']['buildsTime']);
+        $this->assertIsArray($response['body']['buildsMbSeconds']);
         $this->assertIsArray($response['body']['executions']);
         $this->assertIsArray($response['body']['executionsTime']);
+        $this->assertIsArray($response['body']['executionsMbSeconds']);
```

Comment:
We increased count by 6, but only added 2 new assertions

Reply:
The other two are for mbSeconds